advanced/am-pattern-2/templates/am-analytics/dashboard/wso2am-pattern-2-am-analytics-dashboard-deployment.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Chathuru license header needs to be updated.

Reply:
Fix in https://github.com/Chathuru/kubernetes-apim/commit/0205f6311ce5b9ff81ebe108aa79ff7a040078e1 commit